src/engine/renderer/tr_image_jpg.cpp

```diff
@@ -135,10 +122,10 @@ void LoadJPG( const char *filename, unsigned char **pic, int *width, int *height
 	/* Step 2: specify data source (eg, a file) */
 
 #if JPEG_LIB_VERSION < 80
-	jpegfd = fmemopen( fbuffer.b, len, "r" );
+	jpegfd = fmemopen( reinterpret_cast<const unsigned char*>(data.data()), len, "r" );
```

Comment:
You need this not to break things for some code path:
```suggestion
	jpegfd = fmemopen( reinterpret_cast<const unsigned char*>(data.data()), data.size(), "r" );
```

I have an impression that this `#if JPEG_LIB_VERSION < 80` codepath is dumb though, because the biggest libjpeg implementation today is libjpeg-turbo and while it exposes an ABI compatible with libjpeg 6.2 (JPEG_LIB_VERSION == 62), it implements and exposes `jpeg_mem_src` since at least debian oldoldstable.

I've opened https://github.com/DaemonEngine/Daemon/pull/769 for that, you can cherry-pick it here if you want.

Reply:
I fixed it in case we decide not to remove that codepath